site/content/in-dev/unreleased/getting-started/using-polaris.md

```diff
@@ -231,6 +231,8 @@ INSERT INTO quickstart_table VALUES (1, 'some data');
 org.apache.iceberg.exceptions.ForbiddenException: Forbidden: Principal 'quickstart_user' with activated PrincipalRoles '[]' and activated grants via '[quickstart_catalog_role, quickstart_user_role]' is not authorized for op LOAD_TABLE_WITH_READ_DELEGATION
 ```
 
+Additionally, a Getting Started experience for using Spark with Jupyter Notebooks are documented [here](https://github.com/apache/polaris/blob/main/getting-started/spark/README.md).
```

Comment:
```suggestion
Additionally, a Getting Started experience for using Spark with Jupyter Notebooks is documented [here](https://github.com/apache/polaris/blob/main/getting-started/spark/README.md).
```

Reply:
Also, this line looks kind of random more generally. This doesn't seem related to the above despite saying `Additionally,`.

Maybe we can break out a new header called like `Beyond the Quickstart` or `Next Steps` or whatever and put this (and other relevant links) there.